compiler/rustc_hir_typeck/src/fn_ctxt/suggestions.rs

```diff
@@ -794,7 +794,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             return;
         };
 
-        // get all where BoundPredicates here, because they are used in to cases below
+        // get all where BoundPredicates here, because they are used in two cases below
```

Comment:
This is an atypical correction (I don't generally check 2-letter words), the `in to` check flagged this. I'm pretty sure my correction is right.